showcase/openfe_showcase.ipynb

```diff
@@ -2,12 +2,11 @@
  "cells": [
```

Comment:
<p>Ah this wasn't really obvious what we're doing here after going from LomapAtomMapper to KartografAtomMapper.</p><p> </p><p>Could you maybe add some small sub-titles and a brief "we can also use the KartografAtomMapper..." comment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/125/discussion/#comment-1580565516'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>